gui/quickfort.lua

```diff
@@ -2,7 +2,20 @@
 --[====[
 gui/quickfort
 =============
-In-game dialog interface for the `quickfort` script.
+In-game dialog interface for the `quickfort` script. Any arguments passed to
+this script are passed directly to `quickfort`. Invoking this script without
+arguments is equivalent to running ``quickfort gui``.
+
+Examples:
+
+-------------------------------------  --------------------------------------
```

Comment:
Looks like tables only work with `=====`, not `-----` - fixed on my end

Reply:
Argh thank you. I need to find a way to point sphinx to an external scripts dir when building.

Reply:
Not sure if Sphinx has a way to do that. I usually develop in the `scripts` submodule and use [script-paths.txt](https://docs.dfhack.org/en/stable/docs/Core.html#script-paths) to point DFHack to it.